.zuul.d/jobs.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- job:
+    name: awx-ee-build-container-image
```

Comment:
This is the definition of the `awx-ee-build-container-image`? where does it define what commands it runs? Is that inherited from `ansible-build-container-image`? so this is just a parameterization of that job maybe

Reply:
found it!

https://github.com/ansible/ansible-zuul-jobs/blob/c3cbcdffe007a6f7c1c7e4ff1800fa8a59547041/zuul.d/jobs-container-images.yaml#L3

wasn't in ansible/zuul-jobs, but ansible/ansible-zuul-jobs

Reply:
yes, that is right

Reply:
...which goes to `ansible-buildset-registry`

https://github.com/ansible/project-config/blob/be4ef848fea2a95cd91ff15583b33433e624d7f8/zuul.d/jobs.yaml#L233

Reply:
https://zuul-ci.org/docs/zuul-jobs/docker-image.html#create-parent-jobs is more or less the job design of how container builds work.